resolwe_bio/processes/import_data/seq_reads.yml

```diff
@@ -218,3 +218,220 @@
       re-save-file-list fastqc_archive2 "${RW_FASTQC[@]}"
       re-save-list fastqc_url "${FW_FASTQC_URL[@]}"
       re-save-list fastqc_url2 "${RW_FASTQC_URL[@]}"
+
+- slug: basespace-fastq-single
+  name: BaseSpace Reads (single-end)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq
```

Comment:
The tests won't pass until @mzganec updates `rnaseq` Docker image to include all needed tools.

Reply:
https://github.com/genialis/resolwe-bio/pull/481

Reply:
Tag a specific version of `resolwebio/rnaseq` Docker image. Do the same for the paired-end process.